tests/dsp/pdr.cpp

```diff
@@ -2238,12 +2238,20 @@ TEST(EntityAssociationPDR, testAddContainedEntityNew)
 
 TEST(EntityAssociationPDR, testRemoveContainedEntity)
 {
-    pldm_entity* entities = (pldm_entity*)malloc(sizeof(pldm_entity) * 3);
+    pldm_entity* entities = (pldm_entity*)malloc(sizeof(pldm_entity) * 4);
```

Comment:
Can we add more testcases where first and last child gets removed?

Reply:
+1